lib/generators/templates/add_unique_indexes_migration.rb

```diff
@@ -17,11 +17,8 @@ def with_vanity_connection
 
   def change
     with_vanity_connection do
-      remove_index :vanity_experiments, [:experiment_id]
```

Comment:
I'm a bit confused - why did you move these indexes into `add_participants_unique_index_migration.rb` and add the participants index here? Did you mean to do the opposite?

Reply:
ah yes, the additions should have been made to a separate file `add_participants_unique_index_migration.rb` instead of replacing the pre-existing migration.  let me fix that.